Duplicati/Library/Common/IO/SystemIOWindows.cs

```diff
@@ -487,7 +487,15 @@ IFileEntry FileEntryAlphaFS(string p)
                 objs.Add(new FileSystemAccess((FileSystemAccessRule)f));
 
             dict["win-ext:accessrules"] = SerializeObject(objs);
-            dict["win-ext:accessrulesprotected"] = rules.AreAccessRulesProtected.ToString();
+
+            // Only include the following key when its value is True.
+            // This prevents unnecessary 'metadata change' detections when upgrading from
+            //   older versions (pre-2.0.5.101) that didn't store this value at all.

```

Comment:
Sorry for being overly fastidious, but can we remove these extra spaces before the word `older`?

Reply:
sure thing :)

Reply:
Thanks!